plugins/lookup/passwordstore.py

```diff
@@ -227,13 +229,13 @@ def parse_params(self, term):
 
             # Collect pass environment variables from the plugin's parameters.
             self.env = os.environ.copy()
+            self.env['LANG'] = 'en_US.UTF-8'  # make sure to get errors messages in English ("C.UTF-8" not common enough)
```

Comment:
@felixfontein This is to make sure that error messages looked for in `check_output2` are in English.  This isn't a great solution, but I would assume users of a different locale prefer having pinentry in English over missing those errors (*if* they are important, which I don't really know tbh).

Reply:
So, input welcome.

Reply:
@felixfontein I think all concerns should be addressed by now. The only question is if this fix should move to a separate PR? (ultimately, it's unrelated to correcting the "path as password" issue).

Reply:
>One problem is that en_US.UTF-8 might not be a valid locale.

In which case it falls back to C.

> Usually one uses C.UTF-8 or just C for this (since also C.UTF-8 might not be a valid locale; C always is).

Like a wrote in the comment in the code (in case you've read that), C.UTF-8 is not adopted well enough to be an option yet.

>There's a danger though that when using C, UTF-8 input/output might be mangled.

That was exactly my thought process.

Well, I'll split it to a separate PR (or at least remove it from this one). Personally I'll never be affected by this problem, anyway, I just noticed it while reading the code.